caluma/caluma_form/api.py

```diff
@@ -68,3 +68,25 @@ def save_document(
 
     domain_logic.SaveDocumentLogic.update(document, form=form, meta=meta)
     return document
+
+
+def copy_form(
+    source: models.Form,
+    slug: str,
+    name: Optional[str] = "",
+    description: Optional[str] = "",
+    is_published: Optional[bool] = False,
```

Comment:
If you default to empty string or `False`, it's not really an `Optional` type. Just `str`, or `bool`, respectively.

`Optional[YourType]` is short for `Union[NoneType, YourType]`, and does not denote whether the parameter is optional or not